features/admin.console-settings.v1/pages/console-settings-page.tsx

```diff
@@ -50,6 +50,8 @@ const ConsoleSettingsPage: FunctionComponent<ConsoleSettingsPageInterface> = (
 ): ReactElement => {
     const { [ "data-componentid" ]: componentId } = props;
 
+    const [ isCopied, setIsCopied ] = useState(false);
```

Comment:
Let's make the variable name more descriptive.

```suggestion
    const [ isConsoleUrlCopied, setIsConsoleUrlCopied ] = useState<boolean>(false);
```